src/app/location-info/location-info.component.ts

```diff
@@ -76,7 +76,6 @@ export class LocationInfoComponent implements OnInit {
     this.dataService.getLocation(this.cep)
       .then(response => {
         this.location = new Location(response)
-
         this.city = this.location.localidade;
```

Comment:
é legal que tu de uma estudada em ES6, aqui tu poderia trocar por um object destructuring, por exemplo:
```suggestion
const { city, uf } = this.location;
```